spatialmath/box.go

```diff
@@ -352,3 +352,102 @@ func separatingAxisTest(positionDelta, plane r3.Vector, halfSizeA, halfSizeB [3]
 	}
 	return sum
 }
+
+// ToPointCloud converts a box geometry into a []r3.Vector.
+func (b *box) ToPointCloud(options map[string]interface{}) []r3.Vector {
```

Comment:
I think this was briefly discussed earlier, but this should not be a map[string]interface; the methods that operate directly on the Geometry objects should take a `float64` for resolution.

The reason being, that this is not the actual server function which will consume the protobuf request.

The thing that receives the `RenderPointCloudRequest` should be what parses out `options["resolution"]` or uses the default if it is not set.

Reply:
this makes sense. I will implement this change now.